README.md

```diff
@@ -128,6 +129,15 @@ ALTER SESSION SET SCRIPT_LANGUAGES='<LANGUAGE_ALIAS>=localzmq+protobuf:///<bucke
 * Can push Docker images to Docker registries
 * Run tests for you container against an Exasol DB (docker-db or external db)
 
+## Limitations
+
+* Caution with symbolic links inside flavor path: 
```

Comment:
not only flavor path, all in-directories or out-directories (where subdirectories already exists, where exaslct would write to)